packages/platform-sdk/src/coins/network.models.ts

```diff
@@ -17,158 +17,153 @@ export type CoinTransactionTypes =
-	};
-	Fee?: {
-		all?: boolean;
-	};
-	Identity?: {
-		address?: {
-			mnemonic?: boolean;
-			multiSignature?: boolean;
-			publicKey?: boolean;
-			privateKey?: boolean;
-			wif?: boolean;
-			secret?: boolean;
-			validate?: boolean;
-		};
-		publicKey?: {
-			mnemonic?: boolean;
-			multiSignature?: boolean;
-			wif?: boolean;
-			secret?: boolean;
-		};
-		privateKey?: {
-			mnemonic?: boolean;
-			wif?: boolean;
-			secret?: boolean;
-		};
-		wif?: {
-			mnemonic?: boolean;
-			secret?: boolean;
-		};
-		keyPair?: {
-			mnemonic?: boolean;
-			privateKey?: boolean;
-			wif?: boolean;
-			secret?: boolean;
-		};
-	};
-	Ledger?: {
-		getVersion?: boolean;
-		getPublicKey?: boolean;
-		signTransaction?: boolean;
-		signMessage?: boolean;
-	};
-	Link?: {
-		block?: boolean;
-		transaction?: boolean;
-		wallet?: boolean;
-	};
-	Message?: {
-		sign?: boolean;
-		verify?: boolean;
-	};
-	Peer?: {
-		search?: boolean;
-	};
-	Transaction?: {
-		transfer?: SignatureMethods;
-		secondSignature?: SignatureMethods;
-		delegateRegistration?: SignatureMethods;
-		vote?: SignatureMethods;
-		multiSignature?: SignatureMethods;
-		ipfs?: SignatureMethods;
-		multiPayment?: SignatureMethods;
-		delegateResignation?: SignatureMethods;
-		htlcLock?: SignatureMethods;
-		htlcClaim?: SignatureMethods;
-		htlcRefund?: SignatureMethods;
-	};
-	Miscellaneous?: {
-		dynamicFees?: boolean;
-		memo?: boolean;
-		utxo?: boolean;
-	};
-	Derivation?: {
-		bip39?: boolean;
-		bip44?: boolean;
-		bip49?: boolean;
-		bip84?: boolean;
-		secret?: boolean;
-	};
-	Internal?: {
-		fastDelegateSync?: boolean;
-	};
+export type WalletPermission = "read" | "write";
+
+export interface ImportMethod {
+	default: boolean;
+	permissions: WalletPermission[];
 }
 
+export type MnemonicMethod = "bip39" | "bip44" | "bip49" | "bip84";
+
 export interface NetworkManifest {
-	id: string;
+	uuid: string;
 	type: string;
 	name: string;
 	coin: string;
-	explorer: string;
 	currency: {
 		ticker: string;
 		symbol: string;
 	};
-	fees: {
-		type: FeeType;
-		ticker: string;
-	};
-	crypto: {
+	hosts: NetworkHost[];
+	constants: {
 		networkId?: string;
 		blockchainId?: string;
 		assetId?: string;
-		slip44?: number;
+		slip44: number;
 		bech32?: string;
-		signingMethods?: {
-			mnemonic?: boolean;
-			privateKey?: boolean;
-			wif?: boolean;
-		};
-		derivation?: {
-			extendedPublicKey: boolean;
-		};
-		expirationType: ExpirationType;
-	};
-	networking: {
-		hosts: string[];
-		hostsMultiSignature?: string[];
-		hostsArchival?: string[];
 	};
 	governance?: {
-		voting?: {
-			enabled: boolean;
-			delegateCount: number;
-			maximumPerWallet: number;
-			maximumPerTransaction: number;
+		delegateCount: number;
+		votesPerWallet: number;
+		votesPerTransaction: number;
+	};
+	transactions: {
+		expirationType: ExpirationType;
+		types: CoinTransactionTypes[];
+		fees: {
+			type: FeeType;
+			ticker: string;
 		};
+		memo?: boolean;
+		utxo?: boolean;
+	};
+	import: {
```

Comment:
import not part of featureFlags?

Reply:
Feature flags describe the methods that are safe to call without encountering an exception, this is more just information for the client on what is possible with this network.